StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetFormFactory/PaymentSheetFormFactoryConfig.swift

```diff
@@ -8,7 +8,7 @@ import Foundation
 @_spi(STP) import StripePayments
 
 enum PaymentSheetFormFactoryConfig {
-    case paymentSheet(PaymentSheet.Configuration)
+    case paymentSheet(PaymentElementConfiguration) // TODO(porter) Change this back to PaymentSheet.Configuration when implementing embedded showing forms, and add a .embedded(EmbeddedPaymentElement.Configuration) case.
```

Comment:
Interesting, ooc why a new case? Maybe this case is just renamed to `paymentElement`?  Is the form supposed to be different between embedded and PS?